### PR TITLE
Big Data Books

### PR DESCRIPTION
**Pull request by vimoxshah**
Big Data Book list added
[https://github.com/vimoxshah/Learning-Resources/blob/master/BigData/books.md]

### CODE REVIEW

File: BigData/books.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+* [Data Journalism Handbook] (http://datajournalismhandbook.org/1.0/en/) - Jonathan Gray, Liliana Bounegru, and Lucy Chambers - `Beginner`
+* [Building Data Science Teams] (http://assets.en.oreilly.com/1/eventseries/23/Building-Data-Science-Teams.pdf) - DJ Patil - `Beginner`
+* [Information Theory, Inference, and Learning Algorithms] (http://www.inference.phy.cam.ac.uk/itprnn/book.html) - David MacKay - `Intermediate`
+* [Mathematics for Computer Science] (http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-042j-mathematics-for-computer-science-fall-2010/readings/MIT6_042JF10_notes.pdf) - Eric Lehman, Thomas Leighton, and Albert R. Meyer - `Beginner`
+* [The Field Guide to Data Science] (http://www.boozallen.com/media/file/The-Field-Guide-to-Data-Science.pdf) - `Beginner`
+
+
+###MOOCs about Data Science
+* [Data Mining with Weka] (http://www.cs.waikato.ac.nz/ml/weka/mooc/dataminingwithweka/) - Ian H. Witten - `Intermediate`
+* [Introduction to Data Science] (https://class.coursera.org/datasci-001/class) - Bill Howe (Coursera) - `Beginner`
```
Comment (PareshMayani) [inline BigData/books.md]: Courseera and udacity are not the books but online courses. FYI courses should be listed separately in "courses.md" file. For example, https://github.com/GDGAhmedabad/Learning-Resources/blob/master/android/courses.md

**Comment by PareshMayani** [comment]
This is huge list. Just out of curiosity, did you created by yourself or copied from any source?

**Comment by vimoxshah** [comment]
Big data related books added by me and other books list copied from this [(https://github.com/vhf/free-programming-books)] link

**Comment by PareshMayani** [comment]
@PSanni could you please verify this commit? share your comments, if any!

**Comment by PareshMayani** [comment]
My comment, I don't want to prepare a repository with such huge list of copied links but rather those materials/links which we actually refer on a daily basis or a book from which we referred while starting!

**Comment by vimoxshah** [comment]
whatever  books i mentioned in big data that are daily basis used and very hepful for beginners that's my experiance

**Comment by PSanni** [comment]
@vimoxshah 
Hive and Pig are kind a thing which we could include in programming or scripting language, same alike SQL. No doubt both are limited to Hadoop only, but i suggest to have sub section for both under same directory. 

@PareshMayani 
Therefrom everything looks good for me. :+1:

**Comment by PSanni** [comment]
Everything looks similar in all commits, are there anything different @vimoxshah ???

**Comment by vimoxshah** [comment]
As @PareshMayani  suggest that he does not want to long list so at last deleted all other reference books and kept only fundamental books which are very much useful and needs daily basis for beginner.
 In first two commits there was some markdown syntax error. **.sorry for that.** that's my mistake.

**Comment by Brijeshrpatel9** [comment]
@PareshMayani : Big Data is a broad term. But I believe if we can post video tutorial instead of books than it would be much easy for beginners to start working on these technology. I have some suggestions for Apache Spark (next version of Hadoop) as I already worked on Apache Spark in my internship. If we are targeting beginners than we can exclude **Flume** resources. I don't think it will be useful.

**Comment by PareshMayani** [comment]
@Brijeshrpatel9 yes please list down all of those video tutorials and resources!

**Comment by vimoxshah** [comment]
@Brijeshrpatel9 i agree that big data is broad term.but this is nice initiative.
 I think Flume is easy to understand if beginner give some time. Spark is much more difficult for beginner because it requires too much understanding of core of hadoop  and experience also..of course spark is the future you can post videos related to spark or any other topics.

**Comment by PSanni** [comment]
@vimoxshah 
There is no hassle even we are including any advance level tutorial or books, We just need to put it under "Beginner" and "Advance" tags.

**Comment by Brijeshrpatel9** [comment]
@vimoxshah: I have been in touch with big organization that uses all the big data technology. I haven't heard any of them using Flume. On the other hand, Spark is widely accepted technology. I have taken a big data course in my master degree. We have been taught Hadoop and than Spark. I spent 6 months developing my code to run of Apache Spark.

**Comment by PSanni** [comment]
@Brijeshrpatel9 Contradictory, i had few words with my friend working in reputed company, told me that they are using Flume as much as Spark. So, there are multiple possibilities. Better to cover all them. :smile:

**Comment by vimoxshah** [comment]
@Brijeshrpatel9  if you have any good spark tutorial list please add it .because i also want to learn it. i have just basic knowldge of spark.

**Comment by PareshMayani** [comment]
Thank you @vimoxshah for the contribution, but as we discussed please add only links/books/resources which we personally refer on daily basis, rather than sharing a long list!

**Pull request merged**